File: comms/push-notifications/device-registrations/list-device-registrations/list-device-registrations.ts
import { TwilioClient } from "twilio-comms";

async function main() {
    const client = new TwilioClient({
        accountId: "<username>",
        authToken: "<password>",
    });
    await client.pushNotifications.deviceRegistrations.list({
        userId: "comms_pushnotificationuser_01h9krwprkeee8fzqspvwy6nq8",
        appName: "appName",
        pageToken: "pageToken",
        pageSize: 50,
    });
}
main();

File: comms/push-notifications/device-registrations/list-device-registrations/list-device-registrations.rb
require "twilio"

client = Twilio::Client.new(
  account_id: "<username>",
  auth_token: "<password>"
)

client.push_notifications.device_registrations.list(
  user_id: "comms_pushnotificationuser_01h9krwprkeee8fzqspvwy6nq8",
  app_name: "appName",
  page_token: "pageToken",
  page_size: 50
)
